doc/application_plugins/index.rst

```diff
@@ -132,66 +132,35 @@ to the UBAM token, but this is right now not possible with the WebUI.
 
 In the WebUI, test the UBAM token without a Token PIN, test the UBCM token
 with the stored Token PIN, and check the token info afterwards.
-Check the yubikey token via ``/ttype/yubikey``, for example with:
+Check the yubikey token via ``/ttype/yubikey``, for example with::
 
    ykclient --debug --url https://<privacyidea>/ttype/yubikey --apikey "<API key>" "apiid" <otp>
 
 There should be successful authentications (count_auth_success),
 but no failures.
 
 
-.. _freeradius_plugin:
+.. _freeradius:
 
-FreeRADIUS Plugin
------------------
+FreeRADIUS
+----------
 
-If you want to install the FreeRADIUS Plugin on Ubuntu 14.04 LTS this can be
-easily done, since there is a ready made package (see
-:ref:`install_ubuntu_freeradius`).
+Starting with privacyIDEA 2.19, there are two ways to integrate FreeRADIUS:
 
-If you want to run your FreeRADIUS server on another distribution, you
-may download the module at [#rlmPerl]_.
+ * Using a Perl-based privacyIDEA plugin, which is available for FreeRADIUS 2.0.x and above.
+   It supports advanced use cases (such as challenge-response authentication or attribute mapping),
+   but does not implement concurrent handling of multiple requests. Read more about it at :ref:`rlm_perl`.
+ * Using the rlm_rest plugin provided by FreeRADIUS 3.0.x and above. As it implements multithreading,
+   it is suitable for use cases in which high performance is crucial. However, it currently does not support
+   challenge-response or attribute mapping. Read more about it at :ref:`rlm_rest`.
```

Comment:
It also requires setting up freeradius 3, which is not widely shipped with current distributions.

Maybe we should tell, which distris ship with FR3?